dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ByteFormat.java

```diff
@@ -26,6 +26,6 @@ public static String humanReadableByteCount(final long bytes, final boolean flag
         final int exp = (int) (Math.log(bytes) / Math.log(unit));
         final String pre = (flag ? "kMGTPE" : "KMGTPE").charAt(exp - 1) + (flag ? "" : "i");
 
-        return String.format("%.1f %sB", bytes / Math.pow(unit, exp), pre);
+        return String.format("%.2f %sB", bytes / Math.pow(unit, exp), pre);
```

Comment:
Can we consolidate this with the other method so that we don't have the same logic duplicated?